src/rust/engine/graph/src/entry.rs

```diff
@@ -307,12 +307,14 @@ impl<N: Node> Entry<N> {
             context.stats().cleaning_succeeded += 1;
             true
           }
+          // `x` is unused if `test_trace_log`ging is disabled.
```

Comment:
extra g in `ging` part

Reply:
never mind, just realized that it was "logging" with different type from the backticks ...